l1-contracts/test/Registry.t.sol

```diff
@@ -22,40 +22,30 @@ contract RegistryTest is Test {
     assertEq(registry.numberOfVersions(), 1, "should have 1 version");
     DataStructures.RegistrySnapshot memory snapshot = registry.getCurrentSnapshot();
     assertEq(snapshot.rollup, DEAD, "should have dead rollup");
-    assertEq(snapshot.inbox, DEAD, "should have dead inbox");
-    assertEq(snapshot.outbox, DEAD, "should have dead outbox");
     assertEq(address(registry.getRollup()), DEAD);
-    assertEq(address(registry.getInbox()), DEAD);
-    assertEq(address(registry.getOutbox()), DEAD);
 
     vm.expectRevert(abi.encodeWithSelector(Errors.Registry__RollupNotRegistered.selector, DEAD));
     registry.getVersionFor(DEAD);
   }
 
   function testUpgrade() public {
     address newRollup = address(0xbeef1);
-    address inbox = address(0xbeef2);
-    address newOutbox = address(0xbeef3);
-    uint256 newVersion = registry.upgrade(newRollup, inbox, newOutbox);
+    uint256 newVersion = registry.upgrade(newRollup);
 
     assertEq(registry.numberOfVersions(), 2, "should have 2 versions");
     DataStructures.RegistrySnapshot memory snapshot = registry.getCurrentSnapshot();
     assertEq(snapshot.rollup, newRollup, "should have newRollup");
-    assertEq(snapshot.inbox, inbox, "should have inbox");
-    assertEq(snapshot.outbox, newOutbox, "should have newOutbox");
 
     assertEq(address(registry.getRollup()), newRollup);
-    assertEq(address(registry.getInbox()), inbox);
```

Comment:
is it worth keeping these assertions in the test, but just calling the INBOX() on the new rollup?

Reply:
Think it make more sense to do related to #7959 then.